paasta_tools/frameworks/native_scheduler.py

```diff
@@ -138,6 +143,11 @@ def launch_tasks_for_offers(self, driver, offers):
                 operation.launch.task_infos.extend(tasks)
                 driver.acceptOffers([offer.id], [operation])
 
+                for task in tasks:
+                    startup_timer = self.startup_timer_for_task(30, driver, task.task_id.value)
+                    self.starting_tasks[task.task_id.value] = startup_timer
```

Comment:
alternatively we could store staging_timer in MesosTaskParameters 7 lines above? would avoid having to maintain separate dict.